.babelrc

```diff
@@ -1,20 +1,22 @@
 {
-  "presets": ["react", "env"],
+  "presets": [
+    "@babel/preset-react",
+    "@babel/preset-env"
+  ],
   "plugins": [
-    "transform-object-rest-spread",
-    "transform-class-properties",
-    "transform-runtime"
+    "@babel/plugin-proposal-object-rest-spread",
+    "@babel/plugin-proposal-class-properties",
+    [
+      "@babel/plugin-transform-runtime",
+      {
+        "corejs": 2
```

Comment:
Is there a reason why you chose 2 to be the value for corejs?

Reply:
I think it was `npx babel-upgrade --write` that did it. And we're using `core-js` v2 right, so it should be that version right?